src/Components/Cart/Cart.js

```diff
@@ -0,0 +1,23 @@
+import Checkout from "./Checkout";
+import CartItems from "./CartItems";
+import Navbar from "../Nav";
+
+
+const Cart = () => {
+  return (
+    <>
+    <div className="container" id="cart">
+      <Navbar />
```

Comment:
I see that you have imported the `Navbar` component in every page. Since this is repeated in every page, it can be imported once in the `App` component before the `<Routes>` listing.